app/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
+class ShopListRepository(private val shopListApi: ShopListApi) {
```

Comment:
It would be better to split the repository into an interface `ShopListRepository` and an implementation say `ShopListRepositoryImpl` in order to prevent the leaking of implementation details and to improve code testability